internal/net/tcp/option_dialer.go

```diff
@@ -47,11 +47,11 @@ func WithDNSRefreshDuration(dur string) DialerOption {
 		if dur == "" {
 			return
 		}
-		pd, err := timeutil.Parse(dur)
+		_, err := timeutil.Parse(dur)
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blueviolet.svg?style=flat-square)
Is it just working as a validation process?

Reply:
yes